### PR TITLE
Add ubi9:ruby-31 to Ci

### PR DESCRIPTION
**Pull request by jcantrill**
cc @xperimental

### CODE REVIEW

File: ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-ruby31-v1.14.x.yaml [hunk 1]
```diff
@@ -86,7 +86,7 @@ base_images:
   ubi8-ruby31:
```
Comment (xperimental) [inline ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-ruby31-v1.14.x.yaml:86]: Should this be `ubi9-ruby31`? Similarly below where it is used.

Comment (jcantrill) [reply ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-ruby31-v1.14.x.yaml:86]: its just a name but i will change it for correctness

**Pull request merged**